### DISCUSSION TITLE
intType value is rounded for large numbers

### DISCUSSION THREAD

**Discussion by cn-reshma** [graphql-java/graphql-java#2655] [Q&A]
I'm using spring with graphql, I have a large number in my DB and getting the exact value while retrieving in postman `2107377928799868899`
but the same query is returning rounded value while running in graphql page`2107377928799869000`
could anyone help me on fixing this?

**Comment by bbakerman** [comment]
This will be the  scalars you are using.

What is the declaration of the "number field"?

```
type Query {
     numberField : Int
}
```
for example or something else?

**Reply by cn-reshma** [reply]
I am using this integer as part of a JsonNode.

type Query{
         sample (Json){
               numberField : Int
           }
   }

**Reply by bbakerman** [reply]
Right so... there is this library that adds more scalars

https://github.com/graphql-java/graphql-java-extended-scalars

I think you will want either GraphQLLong or GraphQLBigInteger

**Reply by cn-reshma** [reply]
it worked, thank you!
